src/crimson/osd/osd.cc

```diff
@@ -1192,11 +1192,10 @@ seastar::future<> OSD::committed_osd_maps(version_t first,
 seastar::future<> OSD::handle_osd_op(crimson::net::ConnectionRef conn,
                                      Ref<MOSDOp> m)
 {
-  (void) shard_services.start_operation<ClientRequest>(
```

Comment:
Why it is ignored originally...?

Reply:
ms_dispatch message,  seems needn't wait it finish.  let me try if std::ignore works.

Reply:
std::ignore is same to (void), and ms_dispatch() should not be linked with the op's future to block the next dispatch.

I think OSD/shardservice probably needs to implement it's own seastar::gate to guard and track all its own ongoing futures.

Reply:
I can't reproduce it now. will close the PR first then try more test.